docs/enso-protocol.md

```diff
@@ -0,0 +1,14 @@
+Enso protocol consists of pair server and client. They communicate by exchanging
```

Comment:
So enso protocol is a pair server + client? If so, it should not be named protocol because the name "protocol" in no way would tell me that it is a server + client. For me, protocol defiens the way they communicate, not a wrapper around them.